### ISSUE TITLE
Ideas for improving this report

### ISSUE THREAD

**Issue by 1029YanMa** [etc5521-2020/assignment-1-bilby#6]
Firstly, we can improve the previous issues, which was mentioned in the assignment 1 feedback:

1. Modify the table width, making it easier for the readers to read the tables. 
2. Round the percentage output in table 3.1.

Secondly, after recalling the lectures, apply the methods we have learnt to this report:
3. In Q 3.2, combine the "**slides" types together to get a better understanding of the data distribution, like we did on the Olympic data.

Lastly, add some new analysis as supplements. I'd like to do some analysis on state level:
4. Analyse the tx_comp data by state, find the states with the most amusement park injuries. And create a heat map of injuries.
5. Find out the most dangerous ride in each state, displaying the number of injuries and the manufacturer name. This could remind people to avoid the most dangerous ride in their state, and we can further analyse which manufacturer's products caused more injuries over the years.

That's all what I can think of at present. And hope to get a better improvement plan after brain storming with my teammate Weihao.

Yan Ma

**Comment by TengMCing** [comment]
1. Yes, I think the tables need to be fixed. There are multiple issues in the tables, which I also mentioned in my feedback.
2. I agree with you. We can apply techniques of univariate variable visualization and multivariate variables visualization.
3. Yes, we can combine some levels to better represent the difference between devices.
4. I don't fully understand your point 4. What is the purpose of performing such an analysis? And will this analysis address the question you want to answer? If you want to compare the goodness of the measures to prevent injuries in each state, it may not be an accurate representation. Since if one state has a high number of injuries, it could simply because there are more visitors in that state.
5. Given there is no data for the number and category of devices in each state, we can not investigate the difference in accident rate of devices among states. Besides, the different number of accidents among states could simply due to the difference in the number of devices. The second one also suffers from the same problem, we do not know the underlying distribution of the manufacturer's products.

In short, we need to come out with a way to properly interpret the data given our population of interest is F(injured by device A|Vistors) instead of F(injured by device A|injured people)

**Comment by 1029YanMa** [comment]
Hi, Weihao! @TengMCing 

Thanks for the advices. 

On the 4th point, I think what we are going to do is expanding the analysis content, and **providing more information according to the data**. So it should not be limited to the existing analysis dimensions, since the original group members have done quite a lot of work. By the way, this unit is called exploratory data analysis.

For point 5, we have to admit that the data sets are limited, but we can still get more infomation and display the results. As you said, there's no data on the total number of vistors or devices, so what we should do is be careful and explain this in our report, but not to avoid to show what the data tells us. 

Lastly, I want to say that we will also encounter a variety of unsatisfactory data in the real work. I think this is a good opportunity for us to learn how to better use the limited data to deliver more infomation and interpret the results properly.

**Comment by TengMCing** [comment]
Fair enough. We can conduct the analysis to see what we can learn from it.
